test/parameters.jl

```diff
@@ -2,7 +2,10 @@
 
     @testset "Postive" begin
         test_parameter_interface(Positive(5.0))
-        test_parameter_interface(Positive(5.0))
+    end
+
+    @testset "Bounded" begin
+        test_parameter_interface(Bounded(-1.0, -0.1, 2.0))
```

Comment:
It would be really good to test this close to the boundaries to make sure that it doesn't explode, and that there's not a sign error that means it can be very slightly outside of the bounds (as opposed to being very slightly inside the bounds)